### ISSUE TITLE
Replace underscore with lodash?

### ISSUE THREAD

**Issue by Blizzara** [teekkarispeksi/nappikauppa2#100]

**Comment by OpaL2** [comment]
In my opinion we should replace underscore to [lodash/fp](https://github.com/lodash/lodash/wiki/FP-Guide). Main advantage of lodash/fp is function currying and with provided flow wrapper it enables clear and easy to read function chains.

Furthermore lodash/fp provides "cherry picking", so instead of including whole library, only used functions can be imported and helps to reduce build sizes.

**Comment by Blizzara** [comment]
Yeah, function chaining is something where underscore sucks, and lodash would probably look nicer. Feel free to change :)
